flink-core/src/main/java/org/apache/flink/util/FlinkUserCodeClassLoaders.java

```diff
@@ -131,8 +130,8 @@ public static class ParentFirstClassLoader extends FlinkUserCodeClassLoader {
      * delegate is nulled and can be garbage collected. Additional class resolution will be resolved
      * solely through the bootstrap classloader and most likely result in ClassNotFound exceptions.
      */
-    @Internal
-    public static class SafetyNetWrapperClassLoader extends URLClassLoader implements Closeable {
+    private static class SafetyNetWrapperClassLoader extends MutableURLClassLoader
```

Comment:
No need to `implements Closeable` because `URLClassLoader` already `implements Closeable`.